src/config/pragmas/validate/_http.js

```diff
@@ -0,0 +1,74 @@
+let { unique } = require('./_lib')
+
+module.exports = function validateHTTP (http, errors) {
+  if (http.length) {
+    unique(http, '@http routes', errors)
+
+    let methods = [ 'get', 'post', 'put', 'patch', 'delete', 'options', 'head', 'any' ]
+    let validMethod = str => methods.some(m => m === str.toLowerCase())
+    let validPath = str => str.match(/^\/[a-zA-Z0-9/\-:._\*]*$/)
+    http.forEach(route => {
+      let { name, method, path } = route
+      if (!validMethod(method)) errors.push(`Invalid @http method: ${name}`)
+      if (!validPath(path)) {
+        // Can only have letters, numbers, dashes, slashes and/or :params
+        function bads (c) {
```

Comment:
Can we lift this function into global scope? Would save us redefining the function (and its variables) inside the loop, and would tidy up / condense down the core module logic.

Reply:
sgtm!